docs/detections/alerts-view-details.asciidoc

```diff
@@ -18,6 +14,12 @@ The alert details flyout contains these informational tabs:
 * *JSON*: The alert data in JSON format.
 * *Osquery Results*: Results from queries attached to rules display on the *Osquery Results* tab. <<add-osquery-response-action,Learn more.>>
 
+To share an alert, click the **Share alert** link in the upper right corner of the flyout. This gives you a shareable URL that you can share with others or paste into a browser.  
+
+IMPORTANT: The **Share alert** link gives you a direct link to the alert's details flyout. Do _not_ use the general URL in the browser to share the alert. That URL will not work. 
```

Comment:
I would rephrase this to something like:

> The Share alert link gives you a direct link to the alert’s details flyout. Using the general URL in the browser to share the alert can cause inconsistencies when sharing due to relative date ranges and filters that you may currently have set in your browser. The share button is the most reliable way to confirm the alert you want to be shared will be shared with the user without the inconsistencies of the other filters.

This also needs to be added:
> Also, if grouping is enabled when navigating to an alert url, the group must be expanded for the flyout to auto-expand

Reply:
Incorporated feedback in [ce95403](https://github.com/elastic/security-docs/pull/3264/commits/ce954031c28b63d971e0e874f7d40a7a8e8af3c2),